### PR TITLE
Update default Basic HTML WYSIWYG

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Updates the default Basic HTML text editor:

- remove media

- add super/sub script
- add HR
- add strikethrough text formatting
- add special character tool
- add "Remove Format" tool
- update allowed HTML tags list

Note that although the core media/image filters are still included, they are all disabled. I was going to remove them but they will only come back with the next export, and these settings reflect the desired state of the editor.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Low
| Relevant links | JIRA issue, bug reports, security alerts, etc.
RIG-22

### CODE REVIEW

File: ecms_base/config/install/editor.editor.basic_html.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ dependencies:
     - filter.format.basic_html
   module:
     - ckeditor
+_core:
+  default_config_hash: AqlPmO16LvJI4D0Ih6u4GFQIzqr5OnLgAUSjcUGWk2g
```
Comment (pfrilling) [inline ecms_base/config/install/editor.editor.basic_html.yml:9]: Did the clean-config script not remove this?

**Review by pfrilling — CHANGES_REQUESTED** [review]
Looks good. Just double checking the clean-config script was run.

**Review by pfrilling — CHANGES_REQUESTED** [review]
We'll also need an update hook to make these configuration changes on the existing sites.

File: ecms_base/config/install/editor.editor.basic_html.yml (hunk 1 above)
Comment (ben-hamelin) [reply ecms_base/config/install/editor.editor.basic_html.yml]: Whoops! Didn't run it before - just did!

File: ecms_base/ecms_base.profile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Update Basic HTML configuration to match the install config.
+ */
+function ecms_base_update_9001(array &$sandbox): void {
```
Comment (pfrilling) [inline ecms_base/ecms_base.profile:15]: Nice! 

One question, does this also run in the `ecms_acquia` profile? If not, we may want to create a new `ecms_distribution` module and put these updates in a service class that then gets called from both profiles.

I have a feeling we'll be using that service class a lot.

Comment (ben-hamelin) [reply ecms_base/ecms_base.profile:15]: Ok I spun up a new site on DEV, then deployed a test branch of the distribution with this branch specified. Looks to have run! http://updatetest.dev-riecms.acsitefactory.com/admin/config/content/formats/manage/basic_html

**Review by pfrilling — APPROVED** [review]
Looks good 👍

**Pull request merged**